src/Game.js

```diff
@@ -3,26 +3,41 @@ import * as Models from './models'
 
 export default class Game {
   constructor() {
-    this.rows = 15
-    this.cols = 15
+    this.rows = 20
+    this.cols = 10
     this.rubble = []
     this.score = 0
-    this.interval = 300
-    this.startAPiece()
+    this.initPiece()
   }
 
   tick() {
-    this.transactionDo( ()=> this.fallingPiece.fallOne(), ()=> this.fallingPiece.liftOne() )
+    this.transactionDo(
+      ()=> this.fallingPiece.fallOne(),
+      ()=> this.fallingPiece.liftOne() )
     if ( this.fallingPiece.maxRow() === this.rows ) {
-      this.convertToRubble()
+      setTimeout( () => this.convertToRubble(), 200 )
       return this
     }
+    if ( this.nextPositionIsRubble() ) {
+      setTimeout( () => this.checkIfRubble(), 200 )
+    }
+    return this
+  }
 
-    const nextPos = this.fallingPiece.points().map( point => new Models.Point( point.row +1, point.col ))
-    if ( nextPos.some( point => this.rubble.some( rubble => rubble.sameAs( point ) ) ) ) {
+  checkIfRubble() {
+    if ( this.nextPositionIsRubble() ) {
       this.convertToRubble()
     }
-    return this
+  }
+
+  nextPositionIsRubble() {
+    const nextPos = this.fallingPiece.points()
+    .map( point => new Models.Point( point.row +1, point.col ))
+    return nextPos
+      .some( point => this.rubble
```

Comment:
This formatting was a little tough for me to parse - consider reformatting to make it a little bit clearer what's going on here...